Packs/PortScan/Playbooks/Port_Scan_-_Internal_Source.yml

```diff
@@ -0,0 +1,3833 @@
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ''
+      wait: 1
+    view: "{\n  \"position\": {\n    \"x\": 150,\n    \"y\": 1050\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '115':
+    id: '115'
+    taskid: 9bad8d4f-857a-4334-8632-7369666e24e6
+    type: title
+    task:
+      id: 9bad8d4f-857a-4334-8632-7369666e24e6
+      version: -1
+      name: User Enrichment
+      type: title
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#none#':
+      - '116'
+      - '117'
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 1940,\n    \"y\": 160\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '116':
+    id: '116'
+    taskid: 9d955831-be26-427e-8e2e-8b083dff1121
+    type: playbook
+    task:
+      id: 9d955831-be26-427e-8e2e-8b083dff1121
+      version: -1
+      name: Account Enrichment - Generic v2.1
+      description: "Enrich accounts using one or more integrations.\nSupported integrations:\n\
+        - Active Directory"
+      playbookName: Account Enrichment - Generic v2.1
+      type: playbook
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '118'
+    scriptarguments:
+      Username:
+        complex:
+          root: inputs.SourceUsernames
+          transformers:
+          - operator: uniq
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ''
+      wait: 1
+    view: "{\n  \"position\": {\n    \"x\": 1710,\n    \"y\": 320\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '117':
+    id: '117'
+    taskid: 49518608-66cd-43c2-885f-1e2f0f1ac26a
+    type: playbook
+    task:
+      id: 49518608-66cd-43c2-885f-1e2f0f1ac26a
+      version: -1
+      name: Active Directory - Get User Manager Details
+      playbookName: Active Directory - Get User Manager Details
+      type: playbook
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#none#':
+      - '118'
+    scriptarguments:
+      UserEmail: {}
+      Username:
+        complex:
+          root: incident
+          accessor: sourceusername
+          transformers:
+          - operator: uniq
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ''
+      wait: 1
+    view: "{\n  \"position\": {\n    \"x\": 2180,\n    \"y\": 320\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '118':
+    id: '118'
+    taskid: 205a206f-0abd-4863-8110-30f4a80230ed
+    type: regular
+    task:
+      id: 205a206f-0abd-4863-8110-30f4a80230ed
+      version: -1
+      name: Save user info and manager email in incident fields
+      description: Saves the user's email address and display name to the Employee Email and Employee Display Name fields, respectively,
+      and the user's manager email is saved to the Employee Manager Email field.

```

Comment:
```suggestion
      description: "Saves the user's email address and display name to the Employee Email and Employee Display Name fields, respectively,\
      \and the user's manager email is saved to the Employee Manager Email field."
```

Reply:
Try this. I think the commas are not being handled correctly and the yml validation fails